.fttemplates/TS Functional Component/[FTName].tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import styles from './[FTName].styles';
+
+const FTName: React.FunctionComponent = () => (
+  <View>
+    <Text> REPLACE ME</Text>
+  </View>
+);
+
+export default FTName;
```

Comment:
We should consider if default exports is sth that we want to keep or discourage. At MasterClass we avoid them at all cost because of [this](https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad)

Same comment applies for the `index.tsx` file
There I would just do:
```js
export { <FTName | capitalize> } from './[FTName]'
```